modules/security-registries-quay.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-registries.adoc
+
+[id="security-registries-quay_{context}"]
+= Storing containers using Red Hat Quay
+link:https://access.redhat.com/products/red-hat-quay[Red Hat Quay] is an
+enterprise-quality container registry product from Red Hat.
+Development for Red Hat Quay is done through the upstream
+link:https://docs.projectquay.io/welcome.html[Project Quay].
+Red Hat Quay is available to deploy on-premise or through the hosted
+version of Red Hat Quay at link:https://quay.io[Quay.io].
+
+Security-related features of Red Hat Quay include:
+
+* *Time Machine*: Allows images with older tags to expire after a set
+period of time or based on a user-selected expiration time.
+
+* * link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/manage_red_hat_quay/index#repo-mirroring-in-red-hat-quay[Repository mirroring]*: Lets you mirror
+other registries for security reasons, such hosting a public repository
+on Red Hat Quay behind a company firewall, or for performance reasons, to
+keep registries closer to where they are used.
+
+* *Action log storage*: Save Red Hat Quay logging output to link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/manage_red_hat_quay/index#proc_manage-log-storage[Elasticsearch storage]
+to allow for later search and analysis.
+
+* * link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/manage_red_hat_quay/index#quay-security-scanner[Clair security scanning]*: Scan images against a variety of Linux
```

Comment:
Same thing about the asterisks rendering

Reply:
Fixed.